tests/dummy/app/pods/demo/template.hbs

```diff
@@ -1,4 +1,6 @@
-
+<div>
+{{outlet}}
```

Comment:
I'd prefer it in a 50% split on the right and a scroll with the links on the left - that way people don't have to scroll up to see what happened when they click a link